src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml

```diff
@@ -11,14 +11,10 @@
   </EnumType>
 
   <!-- Request Capabilities -->
-
-  <!-- Model validation failed as "The value of the property 'IEdmValueAnnotation.Value' must not be null."
-
   <Term Name="SupportedFormats" Type="Collection(Edm.String)">
     <Annotation Term="Core.Description" String="Media types of supported formats, including format parameters" />
-    <Annotation Term="Core.IsMediaType" />
+    <Annotation Term="Core.IsMediaType" Bool="true" />

```

Comment:
Basically, it's same. The default value for Core.IsMediaType is true in Core vocabulary. I just add the default value to make the validate pass. Besides, it's same as Core vocabulary implementation. Please refer to: 
https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Edm/Vocabularies/CoreVocabularies.xml